src/timeline.ts

```diff
@@ -164,13 +177,30 @@ class Timeline implements IAppView {
           events.fire(AppConstants.EVENT_DATASET_SELECTED_RIGHT, d.item);
 
           isClicked = 0;
-          console.log ('second Click');
+          console.log('second Click');
         }
 
 
       });
 
-    //Create Bars
+    }
+
+    // Hide and Show timeline (line + circles)
+    $('.btn-primary').on('click', function (e) {
```

Comment:
Add additional CSS class `.toggleTimeline` to div and use it as selector for D3 instead of `.btn-primary`